test/specs/modules/Sticky/Sticky-test.js

```diff
@@ -131,13 +131,13 @@ describe('Sticky', () => {
       mockPositions({ bottomOffset: 10, height: 50 })
       wrapperMount(<Sticky {...positions} context={contextEl} onStick={onStick} onUnstick={onUnStick} />)
 
-      wrapper.childAt(1).should.have.style('position', 'fixed')
+      wrapper.childAt(0).childAt(1).should.have.style('position', 'fixed')
```

Comment:
[children() now has slightly different meaning](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning)